### PR TITLE
Fix hyperlink

### PR DESCRIPTION
**Pull request by tonyfujs**
Minor edit to fix hyper link at the bottom of the page.

### CODE REVIEW

File: r/content/arrays.Rmd [hunk 1]
```diff
@@ -185,4 +185,4 @@ Scalar, Array, and ChunkedArray objects.  The returned object will be an Arrow o
 
 ### See also
 
-For a more in-depth discussion of Arrow compute functions, see the section on (using arrow functions in dplyr verbs in arrow)[#Using-arrow-functions-in-dplyr-verbs-in-arrow]
+For a more in-depth discussion of Arrow compute functions, see the section on (using arrow functions in dplyr verbs in arrow)[#Use-arrow-functions-in-dplyr-verbs-in-arrow]
```
Comment (lidavidm) [inline r/content/arrays.Rmd:188]: Thanks for this.

Shouldn't the link syntax be `[]()` not `()[]`? Also, should it link to `tables.html#Use-...`?

Comment (thisisnic) [reply r/content/arrays.Rmd:188]: Good catch @lidavidm

Comment (thisisnic) [inline r/content/arrays.Rmd:188]:
```suggestion
For a more in-depth discussion of Arrow compute functions, see the section on [using arrow functions in dplyr verbs in arrow](#Use-arrow-functions-in-dplyr-verbs-in-arrow)
```

File: r/content/arrays.Rmd [hunk 2]
```diff
@@ -185,4 +185,4 @@ Scalar, Array, and ChunkedArray objects.  The returned object will be an Arrow o
 
 ### See also
 
-For a more in-depth discussion of Arrow compute functions, see the section on (using arrow functions in dplyr verbs in arrow)[#Using-arrow-functions-in-dplyr-verbs-in-arrow]
+For a more in-depth discussion of Arrow compute functions, see the section on [using arrow functions in dplyr verbs in arrow](#Use-arrow-functions-in-dplyr-verbs-in-arrow)
```
Comment (lidavidm) [inline r/content/arrays.Rmd:188]:
```suggestion
For a more in-depth discussion of Arrow compute functions, see the section on [using arrow functions in dplyr verbs in arrow](#use-arrow-functions-in-dplyr-verbs-in-arrow)
```

Comment (lidavidm) [reply r/content/arrays.Rmd:188]: I built it locally and it seems the link is still broken - the autogenerated section reference is case-sensitive.

Comment (thisisnic) [reply r/content/arrays.Rmd:188]: Thanks for checking that!

**Comment by lidavidm** [comment]
Thanks @tonyfujs!

**Pull request merged**